packages/jest-message-util/src/index.ts

```diff
@@ -337,9 +337,9 @@ export const formatResultsErrors = (
 ): string | null => {
   const failedResults: FailedResults = testResults.reduce<FailedResults>(
     (errors, result) => {
-      result.failureMessages
-        .map(checkForCommonEnvironmentErrors)
-        .forEach(content => errors.push({content, result}));
+      result.failureMessages.forEach((item: string) => {
```

Comment:
is the explicit type annotation needed?

Reply:
Linter warned about implicit any type

Reply:
![image](https://user-images.githubusercontent.com/1404810/119088821-90b22c80-ba09-11eb-9413-d21e36de59fd.png)

I'm not seeing any error, let's try without